tweaks to coming_from_processing_java.md

villares opened the pull request:
Jim, please see if you like these tweaks:
- Merging a paragraph on the P-Class section.
- Adding the text of the special comment for imported mode code and small wording changes.

villares committed:
tweaks to coming_from_processing_java.md

- Merging a paragraph on the P-Class section.
- Adding the text of the special comment for imported mode code and other tweaks.

hx2A commented:
Thanks for the PR!

One small request. Can  you change this:

```
[Importing py5 code](https://py5coding.org/content/importing_py5_code.html#importing-imported-mode-code)
```

to this:

```
[](importing_imported_mode_code)
```

They both work but the later is cleaner. I know the former was already there before this PR. I'm trying to clean up the documentation to not include links directly to the py5coding.org domain.

hx2A commented on content/coming_from_processing_java.md:
````diff
@@ -468,7 +466,7 @@ from my_other_file import MyClass, my_function # from other_file.py in the same
 ```
 
-When you are using py5 in *imported mode* you can add a special comment to your other files like explained on this [Importing py5 code](https://py5coding.org/content/importing_py5_code.html#importing-imported-mode-code) section.
+When you are using py5 in *imported mode* with code split in more than one file, you might need to add the special comment, `# PY5 IMPORTED MODE CODE`, to the head of your other files, like explained on this [Importing py5 code](https://py5coding.org/content/importing_py5_code.html#importing-imported-mode-code) section.
````
Neat I can add comments to the PR line by line

Just change the link in this line to this:

```
[](importing_imported_mode_code)
```

hx2A replied:
In theory I can modify your PR but I can't see how to do that.

villares replied:
Hi! I made the change but I don't understand how this can link to a different page (importing_py5_code.html) and then the go to the #importing_imported_mode_code anchor.

hx2A replied:
It has to do with the anchor tag added to the markdown in the other file:

https://github.com/py5coding/py5book/blob/3b4195679f5303921829da3fca8f2977a8b4e642/content/importing_py5_code.md?plain=1#L127

It's one of the fancy markdown things Jupyter book adds for us to use for situations like this.

hx2A replied:
I re-built the site and deployed it to prod.

https://py5coding.org/content/coming_from_processing_java.html

Thanks again for your diligence with the PR!

villares committed:
changing the link as instructed

...but I don't understand how this will reach the other page and then the anchor #importing_imported_mode_code
